### ISSUE TITLE
CVE or any other vulnerability number should be set on the proposal side

### ISSUE THREAD

**Issue by akuleshov7** [saveourtool/save-cloud#2228]
Right now there is NO field for setting CVE or any other number of vulnerability

**Comment by Cheshiriks** [comment]
I don't think it's necessary. We should not allow users to add vulnerabilities from existing databases on their own. We must tighten these bases.
Users add only new vulnerabilities they find. for them we will generate the CVE number ourselves.

**Comment by akuleshov7** [comment]
@Cheshiriks no-no, you don’t get the requirements. THE IDEA of our service is to add existing vulnerabilities and have a mapping to CVE to avoid duplication

**Comment by Cheshiriks** [comment]
Complete in https://github.com/saveourtool/save-cloud/pull/2233
